### ISSUE TITLE
Eden needs to start generating "volumes" and "contentInfo" blocks in EVE config

### ISSUE THREAD

**Issue by rvs** [lf-edge/eden#130]
Starting from 5.7.0 EVE now requires two extra top level blocks in its config for apps to be able to use volumes. E.g.:

```
    "contentInfo": [
        {
            "uuid": "6c57922b-15da-41c0-b50d-370eb914db0b",
            "dsId": "861afe82-0759-4996-b964-f48ca5a4ce22",
            "URL": "xenial-amd64-docker-twoipv4",
            "iformat": 3,
            "sha256": "D2157A3CC854BAB68867C201D62CB297920C3353F3E535325521942D58F0074F",
            "maxSizeBytes": 955252736,
            "siginfo": {},
            "displayName": "xenial-amd64-docker-twoipv4"
        },
        {
            "uuid": "d8831c29-1071-4454-b298-29428ec24f16",
            "dsId": "861afe82-0759-4996-b964-f48ca5a4ce22",
            "URL": "goodImage",
            "iformat": 3,
            "sha256": "394ABA604238C8E65228B42495A828422A0AD209BB5E0AE48A84E5A76512501E",
            "maxSizeBytes": 346817024,
            "siginfo": {},
            "displayName": "goodImage"
        }
    ],
    "volumes": [
        {
            "uuid": "22e245cd-9766-4f62-a88d-be8404ba5a41",
            "origin": {},
            "displayName": "test-zc2-os-instance"
        },
        {
            "uuid": "def855b4-4b2d-4452-9719-892e5b6e5858",
            "origin": {
                "type": 2,
                "downloadContentTreeID": "6c57922b-15da-41c0-b50d-370eb914db0b"
            },
            "maxsizebytes": 3221225472,
            "displayName": "TestAppAssignedCom-sc-supermicro-zc2-2020-06-23T19-09-11_0_m_0"
        },
        {
            "uuid": "df757f7c-82ad-4d2b-9f83-93944dcc57ad",
            "origin": {
                "type": 2,
                "downloadContentTreeID": "d8831c29-1071-4454-b298-29428ec24f16"
            },
            "maxsizebytes": 524288000,
            "displayName": "TestAcls-sc-supermicro-zc2-AI-GoodBundle-2020-06-30T00-51-26_0_m_0"
        }
    ]
```

All parts of Eden that require it (including tests, test harness, pod functionality, etc.) need to be updated to support it.

**Comment by giggsoff** [comment]
Hi, @rvs do you have any documentation about "volumes" and "contentInfo"? 
I try to implement them in my branch https://github.com/itmo-eve/eden/tree/5.7.0 and produce such [config](https://github.com/lf-edge/eden/files/4852456/5.7.0.txt). But apps stale in INITIAL state.

**Comment by mydatascience** [comment]
@rvs ?

**Comment by rvs** [comment]
As I told you guys @giggsoff and @mydatascience -- you need to either dig through what's in the EVE's api folder, look at the zedcloud examples AND jump on a slack channel and have a conversation with Avi.

Can you please do that?

**Comment by rvs** [comment]
This appears to be fixed in #144
